src/utils/coordinateUtils.js

```diff
@@ -402,23 +402,24 @@ export const printHumanReadableCoordinates = (coordinates, coordinateSystem = LV
  *
  * @param {[Number, Number][]} coordinates Coordinates, expressed in EPSG:3857 (metric mercator)
  *   `[[x1,y1],[x2,y2],...]`
+ *   @param {CoordinateSystem} projection The projection used to describe the coordinates
  * @returns {null | CoordinatesChunk[]}
  */
-export const splitIfOutOfLV95Bounds = (coordinates) => {
+export const splitIfOutOfLV95Bounds = (coordinates, projection = WEBMERCATOR) => {
     if (!Array.isArray(coordinates) || coordinates.length <= 1) {
         return null
     }
     // checking that all coordinates are well-formed
     if (coordinates.find((coordinate) => coordinate.length !== 2)) {
         return null
     }
+    let bounds = LV95_BOUNDS
+    if (projection.epsg === WEBMERCATOR.epsg) {
+        bounds = LV95_IN_MERCATOR_BOUNDS
+    }
```

Comment:
Here I would use a switch case that handle both lv95 and webmercator projection and default to return null with eventually an error message to make it clear that the function only supports either lv95 or web mercator

Reply:
Maybe would be worse to also implement other projection like WGS84 anmd lv03